forui/lib/src/widgets/tabs/tabs_style.dart

```diff
@@ -0,0 +1,106 @@
+part of 'tabs.dart';
+
+/// [FTabs]'s style.
+final class FTabsStyle with Diagnosticable {
+  /// The decoration.
+  final BoxDecoration decoration;
+
+  /// The padding.
+  final EdgeInsets padding;
+
+  /// The [TextStyle] of the label
+  final TextStyle unselectedLabel;
+
+  /// The [TextStyle] of the label
+  final TextStyle selectedLabel;
+
+  /// The indicator.
+  final BoxDecoration indicator;
+
+  /// The indicator size.
+  final TabBarIndicatorSize indicatorSize;
```

Comment:
Do we need to expose this? I'm not so sure whether exposing material widgets is a good idea.